oaknational/Oak-Web-Application
# feat/1057 add transcript box

## Description

Add transcript box to the lesson

## Issue(s)

Fixes #1057

## How to test

1. Go to lesson
2. You should see a transcript box at the bottom of the lesson if the lesson has transcript

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
